.github/workflows/e2e-tests.yml

```diff
@@ -28,8 +28,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        containers: [ 1, 2, 3, 4, 5 ]
-    name: App (${{ strategy.job-index }}/${{ strategy.job-total }})
+        containers: [ 1, 2, 3 ]
```

Comment:
reduced amount of jobs since the PR env was struggling. we can try to increase it again later.